localstack/services/s3/provider.py

```diff
@@ -212,14 +212,11 @@ def get_full_default_bucket_location(bucket_name):
 
 class S3Provider(S3Api, ServiceLifecycleHook):
     @staticmethod
-    def get_store(context: Optional[RequestContext] = None) -> S3Store:
-        if not context:
-            return s3_stores[get_aws_account_id()][aws_stack.get_region()]
+    def get_store(account_id: Optional[str] = None, region: Optional[str] = None) -> S3Store:
+        return s3_stores[account_id or DEFAULT_AWS_ACCOUNT_ID][region or AWS_REGION_US_EAST_1]
```

Comment:
note: not sure this is needed, but as it was optional before and I didn't check for all usages, better be safe than sorry for now. The new provider doesn't allow optional values.